src/herder/HerderImpl.cpp

```diff
@@ -844,6 +836,17 @@ HerderImpl::safelyProcessSCPQueueUpToIndex(uint64 nextIndex, bool synchronous)
     }
 }
 
+void
+HerderImpl::lastClosedLedgerIncreased()
+{
+    releaseAssert(isTracking());
```

Comment:
note that in the context of this PR this is fine, but those invariants may not hold in the future:
for example, if we want to be able to "trigger" at the end of catchup without having to wait for an extra ledger.